content/resources/user/gui-submission.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Gen3 - GUI Submission"
+date: 2018-09-12T20:53:20-05:00
+linktitle: /resources/user
+layout: withtoc
+menuname: userMenu
+---
+{{% markdownwrapper %}}
+# GUI Submission of Data Files
+* * *
+
+The following guide details the steps a data contributor must take to submit a project to a Gen3 data commons with the Graphical User Interface (GUI).
+
+* * *
+
+## 1. Prepare Project in Submission Portal
+* * *
+<!--
+This section could be removed if we (semi)automate CMC creation for users
+-->
+
+In order to upload data files, at least one record in the `core_metadata_collection` node must exist. If your project already has at least one record in this node, you can skip to step 2 below.
+
+Do the following to create your first `core_metadata_collection` record:
+
+1. Go to your data commons' submission portal website
+2. Click on 'Submit Data'
+3. Find your project in the list of Projects and click 'Submit Data'
+4. Click 'Use Form Submission' and choose `core_metadata_collection` from the dropdown list (or [edit and upload this TSV](gen3_core_metadata_collection_template.tsv) by clicking 'Upload File' then 'Submit')
+
+    ![node_dropdown.png](node_dropdown.png)
+
+    ![cmc_form.png](cmc_form.png)
+
+
+5. Fill in the required information (see note below)
+6. Click 'Upload submission json from form' and then 'Submit'
+7. Make note of the `submitter_id` of your `core_metadata_collection` record for step 3 below
+
+>__Note:__ Minimally, `submitter_id` and `projects.code` are required properties. The project `code` is the name of your project without the "program-" prefix. For example, if your project URL is https://gen3.datacommons.io/example-training, your project's `code` would be 'training', the `program` would be 'example', and your `project_id` would be the combination: 'example-training'.
+
+You should have received the message:
+
+```
+succeeded: 200
+Successfully created entities: 1 of core_metadata_collection
+```
+
+If you received any other message, then check the 'Details' to help determine the error.
+
+To view the records in the `core_metadata_collection` node in your project, you can go to:
+https://gen3.datacommons.io/example-training/search?node_type=core_metadata_collection
+(replacing the first part of that URL with the URL of your actual project).
+
+## 2. Upload Data Files to Object Storage
+* * *
+
+Data files such as spreadsheets, sequencing data (BAM, FASTQ), assay results, images, PDFs, etc., are uploaded to object storage with the [gen3-client command-line tool](/resources/user/gen3-client).
+
+ >__Note:__ if your data files are already located in cloud storage, such as an AWS or GCS bucket, please see [this page](/resources/user/submit-data/sower) on how to make these files available in a Gen3 data commons.
```

Comment:
OR if you need to support Google.

I think we need a larger disclaimer that this existing Upload Data Files method is very limited in scalability and doesn't work with Google. imo we should push people to use the other method entirely, b/c ideally (once there's better tooling and docs) we should remove the old method to clean up our stack